src/modules/auth/strategies/store-jwt.strategy.ts

```
@@ -0,0 +1,120 @@
+      throwHttpError(ErrorCode.INTERNAL_SERVER_ERROR, {
+        reason: 'JWT_SECRET environment variable is not defined.',
+      });
+    }
+
+    super({
+      jwtFromRequest: ExtractJwt.fromExtractors([
+        (request: any) => {
+          // Utilisation de 'any' ici pour l'objet request d'Express
+          let token: string | null = null;
+          // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+          if (request && request.cookies) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)